### ISSUE TITLE
ABMX issue with modboneimplantor in chara maker and free h (KK)

### ISSUE THREAD

**Issue by Alv4rna** [ManlyMarco/ABMX#42]
When bones implanted with modboneimplantor edited in maker with abmx, it changed properly. But, when the maker is reopen in another session, the abmx edits is not applied immediately. Only after the outfit is changed to another set and back to it again, abmx is applied. This also happened in free h mode, but didn't happen in studio (abmx applied immediately after spawning the character).

**Comment by ManlyMarco** [comment]
Duplicate of #28
